api/app/weather_models/__init__.py

```diff
@@ -55,7 +55,13 @@ def interpolate_between_two_points(  # pylint: disable=invalid-name
     # Create interpolation function.
     function = interp1d(x_axis, y_axis, kind='linear')
     # Use iterpolation function to derive values at the time of interest.
-    return function(xn)
+
+    try:
+        result = function(xn)
+    except Exception as exception:
+        logger.error(exception, exc_info=True)
```

Comment:
Do we still need this logging here? This should be caught and logged higher up no?